src/shimoku_api_python/api/plot_api.py

```diff
@@ -7672,6 +7672,157 @@ def line_with_confidence_area(self,
+            'itemStyle': {
+                'color': 'var(--chart-C2)',
+                'borderRadius': [0, 9, 9, 0],
+            },
+            'emphasis': {
+                'itemStyle': {
+                    'color': 'var(--chart-C2)',
+                    'borderColor': 'var(--chart-C2)',
+                }
+            },
+        }
+
+        neg_dataops_org = {
+            'label': {
+                'position': 'left',
+            },
+            'itemStyle': {
+                'color': 'var(--complementary-red-light)',
+                'borderRadius': [9, 0, 0, 9],
+            },
+            'emphasis': {
+                'itemStyle': {
+                    'color': 'var(--complementary-red-light)',
+                    'borderColor': 'var(--complementary-red-light)',
+                }
+            },
+        }
+
+        options_org = {
+            'toolbox': {
+                'right': 10,
+                'feature': {
+                    'saveAsImage': {},
+                    'dataView': {},
+                }
+            },
+            'tooltip': {
+                'trigger': 'axis',
+                'axisPointer': {
+                    'type': 'shadow'
+                }
+            },
+            'grid': {
+                'top': 80,
+                'bottom': 30
+            },
+            'xAxis': {
+                'type': 'value',
+                'axisLine': {'show': False},
+                'position': 'top',
+                'axisLabel': {'show': False},
+                'splitLine': {
+                    'show': False,
+                }
+            },
+            'yAxis': {
+                'type': 'category',
+                'axisLine': {'show': True},
+                'axisLabel': {'show': False},
+                'axisTick': {'show': False},
+                'splitLine': {'show': False},
+            },
+        };
+
+        seriesopt_org = {
+            'name': 'Revenue',
+            'type': 'bar',
+            'stack': 'Total',
+            'smooth': True,
+            'label': {
+                'show': True,
+                'width': 270,
+                'overflow': 'truncate',
+                'formatter': '{b}\n{c}',
+            },
+        }
+
+        options = deep_update(options_org, options_mod)
+        pos_dataops = deep_update(pos_dataops_org, pos_dataops_mod)
+        neg_dataops = deep_update(neg_dataops_org, neg_dataops_mod)
+        series = deep_update(seriesopt_org, seriesopt_mod)
+
+        series_data_neg = [{
+            'value': val,
+            **neg_dataops,
+        } for val in list(df_neg[neg_col_val])]
+
+        series_data_pos = [{
+            'value': val,
+            **pos_dataops,
+        } for val in list(df_pos[pos_col_val])]
+
+        y_data = list(df_neg[neg_col_name]) + list(df_pos[pos_col_name])
+        options['yAxis']['data'] = y_data
+
+        series_data = series_data_neg + series_data_pos
+        series['data'] = series_data
```

Comment:
I don't think this chart needs to have the data baked in, as it is a horizontal bar chart, it would be better to use the dynamic option, as the other charts do